packages/web-app-user-management/src/components/Groups/GroupsList.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div>
```

Comment:
In theory this wouldn't need a wrapping div since it has only one root node anyways?